src/shared/LogOut/LogOut.tsx

```diff
@@ -9,17 +9,17 @@ export const LogOut = () => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
   const cookies = new Cookies();
   
-  const handleLogOut = (e: React.MouseEvent<HTMLButtonElement>) => {
-    e.preventDefault();
-    cookies.remove('token');
-    window.location.reload();
-  };
+  // const handleLogOut = (e: React.MouseEvent<HTMLButtonElement>) => {
+  //   e.preventDefault();
+  //   cookies.remove('token');
+  //   window.location.reload();
+  // };
 
   return (
     <>
       <div className={`${classes.container} ${isOpen && classes.close}`}>
         {isOpen ? <CloseIcon className={classes.arrow} onClick={() => setIsOpen(!isOpen)}/> : <ChevronLeftIcon className={classes.arrow} onClick={() => setIsOpen(!isOpen)}/>}
-        <button type="button" className={`${classes.logout} clicked`} onClick={handleLogOut}>WYLOGUJ</button>
+        <button type="button" className={`${classes.logout} clicked`} >WYLOGUJ</button>
```

Comment:
Why not Material Button? And where is logout handler? Do you use class and event listener?